src/app/Dashboard/Charts/JFRMetricsChartController.tsx

```diff
@@ -126,7 +126,7 @@ export class ChartController {
     this._stop();
     this._attach = merge(
       merge(
-        this._updates$.pipe(throttleTime(this._settings.chartControllerConfig().minRefresh)),
+        this._updates$.pipe(throttleTime(this._settings.chartControllerConfig().minRefresh * 1000)),
```

Comment:
This `minRefresh` is already in seconds?

Reply:
Right, the stored value is in seconds, but the `throttleTime` function takes milliseconds.

Reply:
ohhh didnt notice that! Thanks :D